lib/venue.rb

```diff
@@ -1,2 +1,20 @@
 class Venue
+  attr_reader :name, :capacity
+  def initialize (name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def patrons
```

Comment:
If you were to add `patrons` to your `attr_reader` you could get rid of this method